aptos-move/move-examples/cli_args/Move.toml

```diff
@@ -1,12 +1,11 @@
+# :!:>manifest
```

Comment:
This is a tag for the docs site code injector, typically used for calling out individual source code sections. In this case I was able to switch to a literal file include since the entire file was tagged.